src/aspire/reconstruction/mean.py

```diff
@@ -43,7 +288,7 @@ def compute_kernel(self):
 
         logger.info("Computing non-centered Fourier Transform")
         kernel = fft.mdim_ifftshift(kernel, range(0, 3))
-        kernel_f = fft.fft2(kernel, axes=(0, 1, 2))
+        kernel_f = fft.fftn(kernel, axes=(0, 1, 2))
```

Comment:
Was this working before?

Reply:
Surprisingly that line was okay, but it probably shouldn't have been, as you noticed.

Reply:
Funny though how my reaction to this line is has stayed nearly constant for ~2 years.